packages/search/src/use-fuzzy-search.ts

```diff
@@ -6,19 +6,12 @@ const defaultOptions = {
 	ignoreLocation: true,
 };
 
-type KeysProp< T > = T extends string
-	? {
-			keys?: never;
-	  }
-	: {
-			keys: Fuse.IFuseOptions< T >[ 'keys' ];
-	  };
-
 export type UseFuzzySearchOptions< T > = {
 	data: T[];
 	options?: Partial< Fuse.IFuseOptions< T > >;
 	query?: string;
-} & KeysProp< T >;
+	keys?: Fuse.IFuseOptions< T >[ 'keys' ];
```

Comment:
The previous type, that disallows the `keys` option when you're just searching a list of strings, still seems to be correct to me.

Was this just a simplification to make things a bit easier to read?

Reply:
It was actually not working, but I've found a way around it.